### PR TITLE
change projjb to mellanox sdk_3 repo

### PR DESCRIPTION
**Pull request by jeniaka**

### CODE REVIEW

**Comment by jeniaka** [comment]
bot:retest

**Comment by jeniaka** [comment]
bot:retest

**Comment by jeniaka** [comment]
bot:retest

File: plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml [hunk 1]
```diff
@@ -60,43 +60,52 @@ steps:
...
+      /etc/init.d/ufmd start
       .ci/do_load_plugin.sh
     parallel: false
 
   - name: add plugin
     agentSelector: "{nodeLabel: 'UFM-POC'}"
-    run: .ci/do_add_plugin.sh 
+    run: |
+      /etc/init.d/ufmd start
+      cd plugins/UFM_NDT_Plugin/
```
Comment (lennybe) [inline plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml:107]: pls use 
systemctl start ufm-enterprise

**Comment by jeniaka** [comment]
bot:retest

**Comment by jeniaka** [comment]
bot:retest

**Comment by amit-gueta** [comment]
bot:retest

**Pull request closed without merging**